autopush/router/gcm.py

```diff
@@ -75,6 +75,7 @@ def _route(self, notification, router_data):
             data['body'] = notification.data
             data['con'] = notification.headers['content-encoding']
             data['enc'] = notification.headers['encryption']
+            data['id'] = notification.version
```

Comment:
Does APNS need a similar id for its nack reporting? Or does it already add that?